RocketControlUnitGUI/src/routes/+page.svelte

```diff
@@ -12,6 +12,18 @@
 	const modalStore = getModalStore();
 
 	const PB = new PocketBase('http://192.168.0.69:8090');
+	
+	PB.authStore.clear();
+	let auth = false;
+
+	onMount(async () => {
+		const email = import.meta.env.VITE_EMAIL;
```

Comment:
do we need to add a blank `.env` into the repo?

Reply:
Nah, if you dont have a .env you just wont be authed